proto/private/prototest/testing.go

```diff
@@ -32,6 +32,7 @@ func AssertDeepEqual(t TestingT, x, y interface{}, opts ...cmp.Option) {
 func AssertElementsMatch[V any](
 	t TestingT, listX, listY []V, opts ...cmp.Option,
 ) {
+	t.Helper()
```

Comment:
This is an intentional stray update, as the lack of this confused some test debugging.